### DISCUSSION TITLE
Does Trino support having multiple coordinators?

### DISCUSSION THREAD

**Discussion by sathishbaskar** [trinodb/trino#15230] [Q&A]
I see that Presto supports [disaggregated coordinators](https://prestodb.io/blog/2022/04/15/disggregated-coordinator) to enable horizontal scaling of coordinators. Is there a similar feature in Trino, either now or planned? 

*Context - I'm exploring options to support a high throughput requirement (>10k TPS) for interactive analytics on big data (>100 PB).*

**Comment by findepi** [comment]
>  high throughput requirement (>10k TPS) 

If this means "more than 10 thousand queries per second", then I'd assume single Trino cluster wouldn't be my primary choice.

> on big data (>100 PB).

You mean each query will be processing more than 100 PB of data?
That would mean 10000 * 100 PB of data scanned a second, so 1 zettabyte per second (if I do the math correctly).
